sample/sample-with-di/helloworldfeature/src/main/java/com/dropbox/kaiken/sample/advancedsample/helloworldfeature/DiSampleUser.kt

```diff
@@ -0,0 +1,13 @@
+package com.dropbox.kaiken.sample.advancedsample.helloworldfeature
+
+import com.dropbox.kaiken.skeleton.core.SkeletonAccessToken
+import com.dropbox.kaiken.skeleton.core.SkeletonUser
+import com.dropbox.kaiken.skeleton.usermanagement.SkeletonMapper
+
+class DiSampleUser(val userId: String, val accessToken: SkeletonAccessToken, val name: String)
+
+class RealSkeletonUserMapper : SkeletonMapper<DiSampleUser> {
```

Comment:
nit you can do 
@ContributeBinding(Scope)

that way you wont need to have the provider later

Reply:
Ah - I thought I could do that but the generics caused dagger / anvil to go haywire.  Let me get the exact error.

Reply:
```
e: com.squareup.anvil.compiler.api.AnvilCompilationException: Back-end (JVM) Internal error: Class com.dropbox.kaiken.sample.advancedsample.helloworldfeature.RealSkeletonUserMapper binds com.dropbox.kaiken.skeleton.usermanagement.SkeletonMapper, but the bound type contains type parameter(s) <T>. Type parameters in bindings are not supported. This binding needs to be contributed in a Dagger module manually.
```

Reply:
maybe localize the binding module to here then?